petab/visualize/cli.py

```diff
@@ -0,0 +1,54 @@
+"""Command line interface for visualization"""
+import argparse
+from pathlib import Path
+import matplotlib.pyplot as plt
+from .plot_data_and_simulation import plot_problem
+from .. import Problem, get_simulation_df, get_visualization_df
+
+
+def _parse_cli_args():
+    """Parse command line arguments"""
+
+    parser = argparse.ArgumentParser(
+        description='Create PEtab visualizations.')
+
+    parser.add_argument('-y', '--yaml', dest='yaml_file_name', required=True,
+                        help='PEtab YAML problem filename')
+    parser.add_argument('-s', '--simulations', dest='simulation_file_name',
+                        required=False,
+                        help='PEtab simulation filename')
+    parser.add_argument('-o', '--output-directory', dest='output_directory',
+                        required=True, help='Output directory')
+    parser.add_argument('-v', '--visualizations', required=False,
+                        dest='visualization_file_name',
+                        help='PEtab visualization specification filename')
+    parser.add_argument('--style', required=False,
+                        dest='style_file_name',
+                        help='Matplotlib style file')
```

Comment:
This is nice, adding an example at some point to show how Seaborn defaults could be used could be helpful

Reply:
e.g., `--style seaborn-colorblind`